File: python/long_sentence_aksk_demo.py
# -*- coding:utf-8 -*-
from ais_sdk.utils import encode_to_base64
from ais_sdk.long_sentence import long_sentence_aksk
from ais_sdk.utils import init_global_env

if __name__ == '__main__':
    # Services currently support North China-Beijing 1 (cn-north-1)
    init_global_env('cn-north-1')

    #
    # access asr, long_sentence,post data by token
    #
    app_key = '*************'
    app_secret = '************'

    demo_data_url = 'https://obs-ch-sdk-sample.obs.cn-north-1.myhwclouds.com/lsr-1.mp3'
    # call interface use the file
    result = long_sentence_aksk(app_key, app_secret, '', demo_data_url)
    print result

    # call interface use the file
    result = long_sentence_aksk(app_key, app_secret, encode_to_base64('data/asr-sentence.wav'))
    print result

File: javascript/long_sentence_aksk_demo.js
/**
 * 长语音识别服务ak，sk请求方式的使用示例
 */
var lsen = require("./ais_sdk/long_sentence");
var utils = require("./ais_sdk/utils");

var app_key = "*************";
var app_secret = "************";

demo_data_url = "https://obs-ch-sdk-sample.obs.cn-north-1.myhwclouds.com/lsr-1.mp3";

var filepath = "./data/asr-sentence.wav";
var data = utils.changeFileToBase64(filepath);

lsen.long_sentence_aksk(app_key, app_secret, "", demo_data_url, "", function (result) {
    console.log(result)
});

lsen.long_sentence_aksk(app_key, app_secret, data, "", "", function (result) {
    console.log(result)
});

